src/app/shared/components/image-form-control/image-form-control.component.ts

```diff
@@ -118,7 +118,39 @@ export class ImageFormControlComponent implements OnInit, ImageFormControlCompon
     }
   }
 
-  public fileChangeEvent(event: string): void {
+  public fileChangeEvent(event: Event): void {
+    const target = event.target as HTMLInputElement;
+    this.imageDecoder(target.files[0], (ev: ProgressEvent<FileReader>) => {
+      const img = new Image();
+      img.src = ev.target.result as string;
+      img.onload = (): void => {
+        const config = this.cropperConfig;
+        if (img.width < config.cropperMinWidth || img.height < config.cropperMinHeight) {
+          this.store.dispatch(
+            new ShowMessageBar({
+              message: SnackbarText.errorForSmallImg,
+              type: 'error'
+            })
+          );
+          this.inputImage.nativeElement.value = '';
+        } else if (img.width > config.cropperMaxWidth || img.height > config.cropperMaxHeight) {
+          this.store.dispatch(
+            new ShowMessageBar({
+              message: SnackbarText.errorForBigImg,
+              type: 'error'
+            })
+          );
+          this.inputImage.nativeElement.value = '';
+        } else {
+          this.openCropperModal(event);
+        }
+      };
+    });
+  }
+
+  public writeValue(_obj: unknown): void {}
```

Comment:
As its method is empty and we're using provided formControl as input property, then, please, investigate and decide whether solution we should follow - implement FormControl itself or use provided one (then remove accessor provider on the top and inherited methods).